### DISCUSSION TITLE
👋 Welcome to Prisma Discussions!

### DISCUSSION THREAD

**Discussion by ludralph** [prisma/prisma#17839] [Announcements]
# **👋 Welcome to Prisma Discussions!**

We are glad to see you! Your feedback plays a crucial role in the future of Prisma.

## 👋 Discussion vs Issue - what to choose?

➡️ When should you open a /prisma GitHub **Discussion**?

- Troubleshooting errors or suspected bugs
- How to do “X” with Prisma ORM ecosystem (Client, Schema, Migrate, Studio)
- Technical discussions about tech stack choices, design decisions, implementation issues, etc.

➡️ When should you open or comment on a /prisma GitHub **Issue**?

- Confirmed Prisma bugs
- [Contributing to the Prisma open source](https://github.com/prisma/prisma/blob/b616335466198c0da15b3aef93db8528c3a1a826/CONTRIBUTING.md)
- [Providing feedback on Preview Features](https://github.com/prisma/prisma/issues/3108)

⚠️ Some best practices to make our Technical Support Engineers super happy:

- **The more information the better** - if you can’t provide a minimal reproduction repository, then add context, environment information, code snippets, steps you’ve taken, screenshots - anything you can! It is all helpful for understanding and identifying the solution faster.
- **Comment, upvote, & follow existing Discussions / Issues / PRs** - do a quick search before opening a new Discussion / Issue / PR - a similar question might already exist.
    
    If Prisma currently doesn't have a certain feature, be sure to check out the [roadmap](https://www.prisma.io/docs/about/prisma/roadmap) to see if this is already planned for the future. We usually link the relevant GitHub issues so that you may leave a +1 and comment with your thoughts!
    
- **Check out our [Docs](http://prisma.io/docs) and the [Prisma Data Guide](https://www.prisma.io/dataguide/)** - we thoroughly document every aspect of our ecosystem, from configuring Prisma to deploying your application.
- **Follow our [Code of Conduct](https://github.com/prisma/prisma/blob/main/CODE_OF_CONDUCT.md)**

If you have questions about the **Prisma Data Platform**, login into into your account and use the in-app messaging feature! 🙏 

Thank you for joining our community, happy coding! 💚

**Comment by ManuelTeranCampi** [comment]


**Reply by janpio** [reply]
Please open a new discussion with your question  this is not the right place to help you. Thanks.
When that happened @ludralph can remove your comment here and lock the discussion so not other accidental comments can happen.
